presidio-image-anonymizer/presidio_image_anonymizer/image_analyzer_engine.py

```diff
@@ -0,0 +1,113 @@
+from typing import List
+from presidio_image_anonymizer.image_recognizer_result import ImageRecognizerResult
+from presidio_image_anonymizer.ocr import OCR
+from presidio_analyzer import RecognizerResult
+from presidio_analyzer import AnalyzerEngine
+
+
+class ImageAnalyzerEngine:
+    """ImageAnalyzerEngine class."""
+
+    def analyze(self, image: object) -> List[ImageRecognizerResult]:
+        """Analyse method to analyse the given image.
+
+        :param image: PIL Image/numpy array or file path(str) to be processed
+
+        :return: list of the extract entities with image bounding boxes
+        """
+        ocr_result = OCR().perform_ocr(image)
+        text = self.get_text_from_ocr_dict(ocr_result)
+
+        analyzer = AnalyzerEngine()
+        analyzer_result = analyzer.analyze(text=text, language="en")
+        bboxes = self.map_entities(analyzer_result, ocr_result, text)
+        return bboxes
+
+    @staticmethod
+    def get_text_from_ocr_dict(ocr_result: dict, separator: str = " ") -> str:

```

Comment:
I was thinking this method could be used outside this class as well for example for an API call where maybe just the OCR text needs to be returned. Alternatively, this could be moved to the ocr class as well. WDYT?

Reply:
I think that putting it in the OCR class is better. This way if a different OCR engine has a different dict it outputs and the logic here is different, one can override it.

Reply:
Changed it now. Let me know what you think